.env

```diff
@@ -26,7 +26,8 @@ REACT_APP_FAQ_URL=https://help.chainsfr.com/en/
 REACT_APP_COINBASE_ACCESS_TOKEN_ENDPOINT=https://g7qwxkbru4.execute-api.us-east-1.amazonaws.com/prod
 REACT_APP_COINBASE_CLIENT_ID=f430f0d8484a4b1d85630b3a43d7c10ec8155a68f4962f4cf3fa3c3e855ff656
 REACT_APP_ETHERSCAN_API_KEY=Y7TBHXI51R6UM7Q47GR2DCCH7X5W4RGYHI
-
+REACT_APP_E2E_TEST_TEST_MAIL_API_KEY=
+REACT_APP_E2E_TEST_TEST_MAIL_NAMESPACE=
```

Comment:
no react_app needed

Reply:
Need it. Otherwise react wont load at npm start